src/util.c

```diff
@@ -520,3 +520,52 @@ int syncRename(const char *oldname, const char *newname)
     fsyncDir(oldname);
     return RR_OK;
 }
+
+static RedisModuleString *zero = NULL;
+
+int RaftRedisExtractBlockingTimeout(RaftRedisCommandArray *cmds, long long *timeout)
+{
+    RedisModule_Assert(cmds->len == 1);
+
+    RaftRedisCommand *cmd = cmds->commands[0];
+    if (cmd->argc < 2) {
+        return REDISMODULE_ERR;
+    }
+
+    if (zero == NULL) {
+        zero = RedisModule_CreateString(NULL, "0", 1);
```

Comment:
Maybe we can create a struct of pointers in RedisRaftCtx and init them RedisRaftCtxInit() then we can use like
```
rr->shared.zero;
rr->shared.one;
```
similar to redis.